### PR TITLE
Check for whitespaces on CI

### PR DESCRIPTION
**Pull request by Luthaf**
Follow-up of #90, adding checks for whitespaces on Travis.

The tests should fail, I will fix all of them when I am sure this makes Travis fail.

### CODE REVIEW

**Comment by Luthaf** [comment]
Ping @g-bauer? Only codecov/patch is failling, because I did not add tests for my whitespaces changes =) 

If you don't have time to review this, tell me and I'll merge it after giving it a second look.

File: scripts/check-whitespaces.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+                "whitespace at the end of the line at {}:{}"
+                .format(path, line_number)
+            )
+    if not lines[-1].endswith("\n"):
+        error("missing new line at the end of the file in {}".format(path))
+
+
+if __name__ == '__main__':
+    for directory in DIRECTORIES:
+        for (root, _, pathes) in os.walk(os.path.join(ROOT, directory)):
```
Comment (g-bauer) [inline scripts/check-whitespaces.py]: s/pathes/paths

**Comment by g-bauer** [comment]
Looks good to me :+1: ! One small typo, but otherwise, I'd merge.

**Comment by Luthaf** [comment]
> One small typo

Fixed!

**Comment by g-bauer** [comment]
Merged! :tada:

**Pull request merged**